RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -224,20 +280,66 @@
 	),
     vertices = 'firstStepGoodPrimaryVertices'
 )
+jetCoreRegionalStepBarrel = jetCoreRegionalStep.clone(
```

Comment:
please move this and `jetCoreRegionalStepEndcap` definitions, which both use `jetCoreRegionalStep` to be done after the modifications to `jetCoreRegionalStep` are complete (line 334 in the current version).
Currently there is a danger of inconsistent modifications or, alternatively, a need for unnecessary repeated modifications.

Reply:
still pending

Reply:
I tried to move afterwards, but it crashed because of not compatible modifier

it does not make much sense, probably, to have the cut based defined --I can agree--
but in case one wants to make use the cut based, then it should be useful, no ?